packages/libraries/yoga/src/index.ts

```diff
@@ -247,5 +252,10 @@ export function useHive(clientOrOptions: HiveClient | HivePluginOptions): Plugin
         }),
       );
     },
+    onDispose() {
+      if (hive[autoDisposeSymbol]) {
+        return hive.dispose();
+      }
```

Comment:
Why do we add this check here?

Reply:
I think it is future proof to have this call. So anything needs to be done during disposal will be done here. Later on, maybe you have some logic for disposal in the Hive client.